src/components/SignInForm.jsx

```diff
@@ -0,0 +1,65 @@
+const SignInForm = () => {
```

Comment:
You've added all the components at once, try not to do this. You should make separate branches

Reply:
Yeah sure